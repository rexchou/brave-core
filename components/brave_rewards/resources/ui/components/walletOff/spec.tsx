/* global jest, expect, describe, it, afterEach */
import * as React from 'react'
import { shallow } from 'enzyme'
import { create } from 'react-test-renderer'
import WalletOff from './index'

describe('WalletOff tests', () => {
  const baseComponent = () => <WalletOff id='off' />

  describe('basic tests', () => {
    it('matches the snapshot', () => {
      const component = baseComponent()
      const tree = create(component).toJSON()
      expect(tree).toMatchSnapshot()
    })

    it('renders the component', () => {
      const wrapper = shallow(baseComponent())
      const assertion = wrapper.find('#off').length
      expect(assertion).toBe(1)
    })
  })
})
