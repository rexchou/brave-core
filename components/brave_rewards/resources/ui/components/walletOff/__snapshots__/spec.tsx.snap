// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WalletOff tests basic tests matches the snapshot 1`] = `
<div
  className="sc-bdVaJa gOxpf"
  id="off"
>
  <div
    className="sc-bwzfXH bmjdvk"
  >
    MISSING: rewardsPanelOffText1
  </div>
  <div
    className="sc-htpNat HUgyh"
  >
    MISSING: rewardsPanelOffText2
  </div>
</div>
`;
